front-end/src/app/(dashboard)/dashboard/page.tsx

```diff
@@ -1,11 +1,248 @@
+                                padding: 3,
+                                borderRadius: 2,
+                                flex: 1,
+                                marginLeft: 1,
+                            }}
+                        >
+                            <Link href="/videos/create" passHref>
+                                <Typography variant="h6" sx={{ cursor: 'pointer' }}>
+                                    <VideoLibraryIcon sx={{ marginRight: 1 }} /> Create New Video
+                                </Typography>
+                            </Link>
+                            <Typography variant="body2" sx={{ marginTop: 1 }}>
+                                Explore video generation options for your projects.
+                            </Typography>
+                        </Box>
+                    </Box>
+
+                    <Box sx={{ display: 'flex', justifyContent: 'space-between', marginTop: 2 }}>
+                        <Box
+                            sx={{
+                                backgroundColor: '#1f1f1f',
+                                padding: 3,
+                                borderRadius: 2,
+                                flex: 1,
+                                marginRight: 1,
+                            }}
+                        >
+                            <Link href="/(dashboard)/images/manage" passHref>
+                                <Typography variant="h6" sx={{ cursor: 'pointer' }}>
+                                    <ImageIcon sx={{ marginRight: 1 }} /> Manage Images
+                                </Typography>
+                            </Link>
+                            <Typography variant="body2" sx={{ marginTop: 1 }}>
+                                View and edit your existing images.
+                            </Typography>
+                        </Box>
+                        <Box
+                            sx={{
+                                backgroundColor: '#1f1f1f',
+                                padding: 3,
+                                borderRadius: 2,
+                                flex: 1,
+                                marginLeft: 1,
+                            }}
+                        >
+                            <Link href="/(dashboard)/videos/manage" passHref>
+                                <Typography variant="h6" sx={{ cursor: 'pointer' }}>
+                                    <VideoLibraryIcon sx={{ marginRight: 1 }} /> Manage Videos
+                                </Typography>
+                            </Link>
+                            <Typography variant="body2" sx={{ marginTop: 1 }}>
+                                Access and modify your video library.
+                            </Typography>
+                        </Box>
+                    </Box>
+
+                   {/* Gallery Section */}
+<Typography variant="h6" sx={{ marginTop: 4 }}>
```

Comment:
Need formatting. run "npm i" in root folder for later or install preitter. v

Reply:
will do later

Reply:
done